ecs-init/config/common.go

```diff
@@ -99,3 +111,42 @@ func DockerUnixSocket() (string, bool) {
 	// eg: Docker daemon restart
 	return "/var/run", false
 }
+
+// Find AZ information from Metadata. If error return a blank result
+func EC2MetadataAZ () string {
+	sessionInstance := session.Must(session.NewSession())
+	metadata := ec2metadata.New(sessionInstance)
+	azName, err := metadata.GetMetadata("placement/availability-zone")
+
+	if err != nil {
+		return ""
+	}
+
+	return azName
+}
+
+//convert AZ name to Region name or default if blank is returned
+func AZToRegionName (azName string) string {
```

Comment:
Same as above:

1. `AZToRegionName ` is global scoped. It doesn't need to be
2. The comment needs to start with the method name as per my previous comment